{
  "id": 5600846,
  "key": "MM-T2697",
  "name": "LDAP User Filter error",
  "createdOn": "2020-06-01T17:46:17Z",
  "objective": null,
  "precondition": null,
  "estimatedTime": null,
  "labels": ["se-prod", "ldap"],
  "customFields": {
    "Playwright": null,
    "Authors": "",
    "Priority P1 to P4": "P4 - Low-Impact (Edge or unsuitable to repeat?)",
    "Update notes": "",
    "Rainforest": ["N/A"],
    "Team Ownership": ["Suite Users"],
    "Manual Test Environments": [],
    "Detox": null,
    "Cypress": "To Do",
    "MMCTL": null,
    "Last Updated": "",
    "Tags": [],
    "Location": "AD LDAP"
  },
  "componentName": null,
  "priorityName": "Low",
  "statusName": "Active",
  "folderName": "AD LDAP",
  "folderFullPath": "suite/ad-ldap",
  "steps": [
    {
      "inline": {
        "description": "LDAP User Filter error<br>————————————————————————————<ol><li>AD/LDAP Settings ➜ User Filter</li><li>Enter:<br>(&amp;(objectCategory=Person)(!(UserAccountControl:<br><br></li><li>Save</li></ol>",
        "testData": null,
        "expectedResult": "<ul><li>Error message shows above save button, \"Invalid AD/LDAP filter\"</li><li>Set back to:<br>(&amp;(objectCategory=Person)(!(UserAccountControl:1.2.840.113556.1.4.803:=2)))<br>or just switch to a different page and back again and verify it returned to previous value</li></ul>",
        "customFields": {
          "Fix Versions": [],
          "Related ticket(s)": "",
          "Important notes about this step": ""
        }
      },
      "testCase": null
    }
  ],
  "caseHashed": "02792195fcec1feba8be6cb0d93c744ec988e576f10b55c184fe005bd1271aea9a6a9eb016559406d3cd923e1644071c",
  "stepsHashed": "ba70722868f0931b5ccb57ae950d82228adcdee78d11ce41edf7545c41524cb92e6f81e869a40d05388ec97eeab740d1"
}
